pyplanet/apps/contrib/jukebox/views.py

```diff
@@ -262,11 +262,12 @@ async def action_jukebox(self, player, values, map_info, **kwargs):
 		await self.app.add_to_jukebox(player, await self.app.instance.map_manager.get_map(map_info['uid']))
 
 	async def action_folders(self, player, values, **kwargs):
-		await self.app.folder_manager.display_folder_list(player)
+		if player.level > player.LEVEL_PLAYER:
+			await self.app.folder_manager.display_folder_list(player)
```

Comment:
This will prevent non-admin/non-operators from showing folders right?, that's not what we want I think.

Reply:
That's not what we want indeed, think this one shouldn't be merged.